test/integration/invalid-arguments.spec.js

```diff
@@ -3,18 +3,20 @@
 var invokeMocha = require('./helpers').invokeMocha;
 
 describe('invalid arguments', function() {
-  it('should exit with failure if arguments are invalid', function(done) {
-    invokeMocha(
-      ['--ui'],
-      function(err, result) {
-        if (err) {
-          return done(err);
-        }
-        expect(result, 'to have failed');
-        expect(result.output, 'to match', /not enough arguments/i);
-        done();
-      },
-      {stdio: 'pipe'}
-    );
+  describe('when argument is missing required value', function() {
```

Comment:
Said this before and provided code for it.
This should be run for _all_ arguments expecting a value, not just `--ui`.